components/brave_wallet_ui/utils/web3-utils.ts

```diff
@@ -20,3 +21,16 @@ export const weiToEther = (
   return new Amount(weiValue)
     .divideByDecimals(assetDecimals)
 }
+
+/**
+ * Converts a lamport-denominated value to a SOL denominated value
+ * Lamport → SOL
+ * @param lamports coins value formatted in it's lowest denomination
+ * @returns
+ */
+export const lamportsToSol = (
```

Comment:
Should we have a test for this?

Reply:
tests added: https://github.com/brave/brave-core/compare/ca6f996896732aaf6b0ae9ab899d4f79de5c0683..5e82502821e64824d7567df4cf916945322d2646